frontend/www/js/omegaup/components/problem/SettingsSummaryV2.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div>
-    <h3 data-problem-title class="text-center mb-4">
+    <h3 :data-problem-title="problem.alias" class="text-center mb-4">
+      <slot name="problemset-title"></slot>
       {{ title }}
```

Comment:
se puede evitar la concatenación y mejor usar una cadena de traducción para desplegar tanto el título como el nombre del concurso?

```suggestion
      <slot name="problemset-title">{{ title }}</slot>
```

Reply:
A ver si está bien como lo hice.

Reply:
sí, mucho mejor!